### PR TITLE
Update System v1.1

### PR DESCRIPTION
**Pull request by JeanTracker**
Add device's charging field in context for sending the device's
charging information to the Play.
System agent query information about battery to the device.

Signed-off-by: Hyojoong Kim <hyojoong.kim@sk.com>

### CODE REVIEW

File: include/interface/capability/system_interface.hh [hunk 1]
```diff
@@ -74,6 +74,20 @@ public:
      * @param[in] reason reason for revoke
      */
     virtual void onRevoke(RevokeReason reason) = 0;
+
+    /**
+     * @brief SDK request information about device's charging status
+     * @param[out] charge device's charging status
+     * @param[out] return device charge status accessible
```
Comment (webispy) [inline include/interface/capability/system_interface.hh]: Please change `@param[out]` to `@retval`, `@return`.

File: core/capability/system_agent.cc [hunk 2]
```diff
@@ -95,8 +94,25 @@ void SystemAgent::updateInfoForContext(Json::Value& ctx)
...
+            if (battery <= 0)
+                battery = 1;
+
+            if (battery > 100)
+                battery = 100;
+
+            root["battery"] = battery;
+        }
+
+        nugu_error("charging: %d, battery: %d", charging, battery);
```
Comment (webispy) [inline core/capability/system_agent.cc]: Please change `nugu_error()` to `nugu_dbg()` or `nugu_info()`.

File: include/interface/capability/system_interface.hh (hunk 1 above)
Comment (JeanTracker) [reply include/interface/capability/system_interface.hh]: Update comment from `@param[out]` to `@return`

File: core/capability/system_agent.cc (hunk 2 above)
Comment (JeanTracker) [reply core/capability/system_agent.cc]: Updated. Thanks.

**Pull request merged**